src/main/java/com/google/devtools/build/lib/remote/ExperimentalGrpcRemoteExecutor.java

```diff
@@ -198,6 +198,11 @@ ExecuteResponse waitExecution() throws IOException {
         Iterator<Operation> operationStream = waitExecutionFunction.apply(request);
         return handleOperationStream(operationStream, /* waitExecution= */ true);
       } catch (StatusRuntimeException e) {
+        if (e.getStatus().getCode() == Code.NOT_FOUND
```

Comment:
Since this is getting more complicated, please add a comment here explaining why we check the NOT_FOUND here (maybe copy&paste words from the PR description).

Reply:
Agreed. Clarified the state and behavior of nextDelayMilis as well here in the case of the handleOperationStream internal NOT_FOUND throw.